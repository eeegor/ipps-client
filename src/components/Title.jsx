import React from 'react';
import './Title.scss';

export const Title = props => {
  const { text } = props;
  return <h1 className="title">{text}</h1>;
};
