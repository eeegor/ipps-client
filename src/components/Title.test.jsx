import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import { Title } from './Title';

describe('Title', () => {
  it('renders markup correctly', () => {
    const wrapper = <Title>Hello</Title>;
    const tree = renderer.create(wrapper).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should adopt a text', () => {
    const wrapper = shallow(<Title text="custom-text">Hello</Title>);
    expect(wrapper.contains('custom-text')).toEqual(true);
  });
});
