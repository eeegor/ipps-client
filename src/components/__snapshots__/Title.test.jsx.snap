// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders markup correctly 1`] = `
<h1
  className="title"
/>
`;
